lib/screenplay/config/places_and_screens.ex

```diff
@@ -1,15 +1,60 @@
 defmodule Screenplay.Config.PlaceAndScreens do
-  @moduledoc false
+  @moduledoc """
+  Module used to defined struct for screen configs stored in places_and_screens.json.
```

Comment:
```suggestion
  Module used to define struct for screen configs stored in places_and_screens.json.
```
Also in the 2 sub-modules' `@moduledoc`s